classwork5/classwork5/SignInViewController.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+
+class SignInViewController: UIViewController {
+    private let username: String = "alsu"
+    private let password: String = "1234"
+    
+    private lazy var loginTF: UITextField = {
```

Comment:
Так лучше не сокращать названия
```suggestion
    private lazy var loginField: UITextField = {
```